miner.go

```diff
@@ -38,3 +38,29 @@ func ParseFile(fileName string, analytics *Analytics) {
 		fmt.Printf("Skipped %d non-JSON lines in '%s'.\n", skippedLines, fileName)
 	}
 }
+
+func ParseAndConvertFile(fileName string, converter *Converter) {
```

Comment:
In a later change, we could align the `Converter` and `Analtyics` interfaces, so we wouldn't need to create two separate methods